shared-bindings/keypad/KeyMatrix.c

```diff
@@ -146,8 +146,8 @@ STATIC void check_for_deinit(keypad_keymatrix_obj_t *self) {
 
 //|     def reset(self) -> None:
 //|         """Reset the internal state of the scanner to assume that all keys are now released.
-//|         Any key that is already pressed at the time of this call will therefore immediately cause
-//|         a new key-pressed event to occur.
+//|         Any key that is already pressed at the time of this call will therefore cause
+//|         a new key-pressed event to occur on the next scan.
```

Comment:
@jepler Did your recent fix make this happen immediately now?

Reply:
Oh it does, I missed that !